common/config/config.go

```diff
@@ -72,32 +72,32 @@ type CRCArbiterInfo struct {
-	HttpRestPort         int                  `json:"HttpRestPort"`
-	MinCrossChainTxFee   int                  `json:"MinCrossChainTxFee"`
-	RestCertPath         string               `json:"RestCertPath"`
-	RestKeyPath          string               `json:"RestKeyPath"`
-	HttpInfoPort         uint16               `json:"HttpInfoPort"`
-	HttpInfoStart        bool                 `json:"HttpInfoStart"`
-	HttpWsPort           int                  `json:"HttpWsPort"`
-	HttpJsonPort         int                  `json:"HttpJsonPort"`
-	NodePort             uint16               `json:"NodePort"`
-	PrintLevel           uint8                `json:"PrintLevel"`
-	MaxLogsSize          int64                `json:"MaxLogsSize"`
-	MaxPerLogSize        int64                `json:"MaxPerLogSize"`
-	MaxTxsInBlock        int                  `json:"MaxTransactionInBlock"`
-	PowConfiguration     PowConfiguration     `json:"PowConfiguration"`
-	VoteHeight           uint32               `json:"VoteHeight"`
-	Arbiters             []string             `json:"Arbiters"`
-	EnableArbiter        bool                 `json:"EnableArbiter"`
-	ArbiterConfiguration ArbiterConfiguration `json:"ArbiterConfiguration"`
-	RpcConfiguration     RpcConfiguration     `json:"RpcConfiguration"`
-	HeightVersions       []uint32             `json:"HeightVersions"`
-	VoteStartHeight      uint32               `json:"VoteStartHeight"`
+	ActiveNet             string               `json:"ActiveNet"`
+	Magic                 uint32               `json:"Magic"`
+	FoundationAddress     string               `json:"FoundationAddress"`
+	Version               int                  `json:"Version"`
+	SeedList              []string             `json:"SeedList"`
+	HttpRestPort          int                  `json:"HttpRestPort"`
+	MinCrossChainTxFee    int                  `json:"MinCrossChainTxFee"`
+	RestCertPath          string               `json:"RestCertPath"`
+	RestKeyPath           string               `json:"RestKeyPath"`
+	HttpInfoPort          uint16               `json:"HttpInfoPort"`
+	HttpInfoStart         bool                 `json:"HttpInfoStart"`
+	HttpWsPort            int                  `json:"HttpWsPort"`
+	HttpJsonPort          int                  `json:"HttpJsonPort"`
+	NodePort              uint16               `json:"NodePort"`
+	PrintLevel            uint8                `json:"PrintLevel"`
+	MaxLogsSize           int64                `json:"MaxLogsSize"`
+	MaxPerLogSize         int64                `json:"MaxPerLogSize"`
+	MaxTxsInBlock         int                  `json:"MaxTransactionInBlock"`
+	PowConfiguration      PowConfiguration     `json:"PowConfiguration"`
+	EnableArbiter         bool                 `json:"EnableArbiter"`
+	ArbiterConfiguration  ArbiterConfiguration `json:"ArbiterConfiguration"`
+	RpcConfiguration      RpcConfiguration     `json:"RpcConfiguration"`
+	OutputHashCheckHeight uint32               `json:"VoteStartHeight"`
```

Comment:
This is the CheckAddressHeight where start to prevent transfer to no-ELA address.
And also change `json:"VoteStartHeight"` to `json:"CheckAddressHeight"`